templates/lb-fargate-service/manifest.yml

```diff
@@ -14,9 +14,10 @@ image:
   port: {{.Image.Port}}
 
 http:
-  # Requests to this path will be forwarded to your service.
+  # Requests to this path will be forwarded to your service. 
+  # To match all requests you can use the "/" path. 
   path: '{{.Path}}'
-  # # You can specify a custom health check path. The default is "/"
+  # You can specify a custom health check path. The default is "/"
   # healthcheck: '{{.HealthCheckPath}}'
```

Comment:
We don't need to fix this? For now it seems like if the users they don't uncomment this one the application fails at the deploy stage, CFN error will be healthcheck path string length < 0.

Reply:
I fixed it by this: https://github.com/aws/amazon-ecs-cli-v2/pull/748/files#diff-5688258a7b2bb57222514d427436e902R55

Now, if there's nothing in the manifest, it defaults to what's in our "default"

Reply:
OH I see :+1:.